yaptide/admin/simulators.py

```diff
@@ -343,6 +357,52 @@ def download_topas_from_s3(path: Path,
     click.echo(f"Installed Geant4 files into {geant_files_path}")
     return True
 
+def download_fluka_from_s3(path: Path,
```

Comment:
This seems duplicating part of the same code as for SHIELD-HIT12A and TOPAS. Can you extract common code to some separate functions ?

Fluka is also proprietary code, we should make some additional protections in case S3 credentials would leak.
I recommend to encrypt Fluka binaries in the same way as we did for SHIELD-HIT12A